knip.jsonc

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://unpkg.com/knip@5/schema.json",
+  "$schema": "https://cdn.jsdelivr.net/npm/knip@5/schema.json",
```

Comment:
Even if we decide to use an alternative CDN, I recon we can skip jsDelivr here since this is a schema mostly for IDE autocompletion.

Reply:
Yes - taking this opportunity to clean up some cruft would be nice.